### PR TITLE
Change group lead link from org view page

### PR DESCRIPTION
**Pull request by liampan**
Only appears for orgType Group 

//TODO welsh

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -729,6 +729,7 @@ partnership.removePartner.caption = Partners in the partnership
 amend.organisation.title = Your organisation’s details
 amend.individual.title = Your details
 amend.group.title = Representative member’s details
+amend.partnership.changeGroupLead.link = Change the representative member of the group
```
Comment (PSouthern) [inline conf/messages.en:732]: can this be `amend.group.changeGroupLead.link` please

File: test/views/amendment/AmendRegistrationViewSpec.scala [hunk 2]
```diff
@@ -100,6 +100,17 @@ class AmendRegistrationViewSpec extends UnitViewSpec with Matchers {
             }))
           }
 
+          "display the change-group-lead-link correctly" in {
+            val maybeLink = Option(view.select("#change-group-lead-link").get(0))
+            organisationType match {
+              case Organisation =>
+                maybeLink mustBe defined
+                maybeLink.get.text() mustBe messages("amend.partnership.changeGroupLead.link")
```
Comment (PSouthern) [inline test/views/amendment/AmendRegistrationViewSpec.scala:108]: can this be `amend.group.changeGroupLead.link` please

File: app/views/amendment/amend_registration_page.scala.html [hunk 3]
```diff
@@ -56,6 +58,18 @@
 
     @pageHeading(messages(titleMessageKey()))
 
+
+    @{registration.registrationType match {
+        case Some(GROUP) =>
+            paragraph(link(
+                messages("amend.partnership.changeGroupLead.link"),
```
[the remark this answers is not in the document]
Comment (liampan) [reply app/views/amendment/amend_registration_page.scala.html:65]: good spot, thatll teacg me to copy and paste 🙈

[the remark this answers is not in the document]
Comment (liampan) [reply app/views/amendment/amend_registration_page.scala.html:65]:
```suggestion
                messages("amend. group.changeGroupLead.link"),
```

File: conf/messages.en (hunk 1 above)
Comment (liampan) [reply conf/messages.en:732]:
```suggestion
amend. group.changeGroupLead.link = Change the representative member of the group
```

File: test/views/amendment/AmendRegistrationViewSpec.scala (hunk 2 above)
Comment (liampan) [reply test/views/amendment/AmendRegistrationViewSpec.scala:108]:
```suggestion
                maybeLink.get.text() mustBe messages("amend.group.changeGroupLead.link")
```

**Review by muninderadavelly — APPROVED** [review]

**Review by PSouthern — APPROVED** [review]

**Pull request merged**